src/doc/src/reference/config.md

```diff
@@ -335,6 +335,13 @@ r = "run"
 
 Aliases are not allowed to redefine existing built-in commands.
 
+Aliases are recursive:
+
+```toml
+rr = "run --release"
+space_example = "rr -- \"command list\""
+```
```

Comment:
Could you provide a more real-world example for `space_example`?

Reply:
@epage Sure, I just took the one from the `config.toml` shown in the docs. Would something like this qualify?

```
toml
rr = "run --release"
space_example = "rr --example example_from_space"
```

Reply:
Ah, if its pre-existing, then thats fine.

How about we also update that earlier config to be recursive?  That way we people who just see that don't read the wrong things into it